integrationtests/csi_api_gen_test.go

```diff
@@ -0,0 +1,109 @@
+package integrationtests
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/kubernetes-csi/csi-proxy/cmd/csi-proxy-api-gen/generators"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// This tests API generator; more specifically, its main goal is to ensure
+// that the API generator works as expected when creating a new API group.
+// On top of this, the regular build checks that all checked-in generated files
+// are up-to-date (i.e. consistent with the current generator).
+
+// TestNewAPIGroup tests that bootstraping a new group works as intended.
+func TestNewAPIGroup(t *testing.T) {
```

Comment:
This test case is having a doc.go? Maybe it is not that important, just want to check is there other test cases that are interesting to us?

Reply:
Yes it does have one. Testing API groups from canonical locations will be done by re-generating all auto-generated files as part of all builds.